dbadmin/testing_data.sql

```diff
@@ -0,0 +1,63 @@
+-- ----------------------------------------------------------------------------
+-- VULN4OPENSHIFT DB - Development Data Insert
+--
+-- This file inserts some data for initial development to use.
+-- This is intended to be temporary until we get full flow working with
+-- real data.
+-- ----------------------------------------------------------------------------
+INSERT INTO cluster (id, uuid, status, type, version, provider, account_id, cve_cache_critical, cve_cache_important, cve_cache_moderate, cve_cache_low) VALUES
```

Comment:
there is no type column in schema currently, maybe it's a mistake, but it's still unclear which columns will be needed and which not